src/mocks/handlers.ts

```diff
@@ -21,6 +22,46 @@ export const handlers = [
     );
   }),
 
+  /**
+   * Build a fake ensure user function
+   * Currently will ALWYAS return Brenda Wedding
+   * To update this, we'll need to track users and user ID's
```

Comment:
Does the people picker sample data have an ID we can use?  Or from which we can derive one?

Reply:
Not sure I understand how hashing works, but in my mind it might have been simpler to just handle it similar to SharePoint, and just give it the next ID based on the length of the array.  You never delete a user from the array, so wouldn't have to worry about ID reuse.